examples/data_staging/io_staging_pipeline.py

```diff
@@ -143,20 +143,20 @@ def unit_state_cb (unit, state):
 
     except Exception as e:
         # Something unexpected happened in the pilot code above
-        print "caught Exception: %s" % e
+        print( "caught Exception: %s" % e)
         raise
```

Comment:
From here and for all the following `print` statements, should we delete the space after the opening `(`?